install.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Install
+order: 30
+---
+
+# Install or upgrade
+
+mozregression is a Python (currently 2.7) package installable via pip. You can
+follow the OS-specific instructions to install pip if you don't know how to do it:
+
+- **Windows**:
+  Download and install python 2.7 from
+  [ActiveState](http://www.activestate.com/activepython/downloads).
+  It will include pip and set python in your PATH.
```

Comment:
Do you think this is a better approach than the previously-suggested use of mozilla-build?